src/plugins/crypto/crypto.c

```diff
@@ -197,6 +298,11 @@ static int elektraCryptoDecrypt (Plugin * handle ELEKTRA_UNUSED, KeySet * data E
 	ksRewind (data);
 	while ((k = ksNext (data)) != 0)
 	{
+		if (!isMarkedForEncryption (k) || isSpecNamespace (k))
+		{
+			continue;
+		}
```

Comment:
No! Thx for pointing it out. Fixed in 9754ff27adad30b45002b6864b0180ce32d34624 .